.github/workflows/pre-release-tests.yml

```diff
@@ -14,11 +14,11 @@ jobs:
     strategy:
       matrix:
         # Current go.mod version and latest stable go version
-        go: [1.16, 1.17]
```

Comment:
Same, can it be in another PR? It helps to provide a clear changelog to users, and for the review too